src/controllers/BrowserControllerFactory.ts

```diff
@@ -1,5 +1,5 @@
 import * as puppeteer from 'puppeteer';
-import playwright from 'playwright-core';
+import playwright from 'playwright-chromium';
```

Comment:
why this change?

Reply:
Because I have upgraded **playwright-chromium** to the newest version.
And in the new **playwright-chromium** package there are other dependencies (without **playwright-core** package).
And **playwright-chromium** has the same typescript types as **playwright-core**.